README.md

```diff
@@ -1 +1,69 @@
-# TodoBackend
+Project Name: 
+`Todolist backend`
+
+Description: 
+`This is a backend application that is developed to provide api's for todolist frontend application`
+
+Application type: 
+`Spring boot application`
+
+Platform used: 
+`Ubuntu 16.04.6 LTS`
+
+IDE used: 
+`Spring Tool Suite 4`
+`Version: 4.4.0.RELEASE`
+
+Tools & Framework used: 
+`Java spring boot framework,`
+`JDK 1.8`
+
+Dependencies:
+
+`1.Spring web`
+
+`2.H2 database`
+
+`3.Spring data JPA`
+
+Database used: 
+`h2 in memory database`
+
+Installation: 
+`To run this app follow given steps,`
+
+Steps for STS4: 
+        
+        1.Clone this repository or download zip in your system.
+        
+        2.After cloning or downloading open this directory in your IDE (STS 4)
+                go to file->import project from file system or archive
+                ->import source 
+                
+        3.After importing project right click on you project
+                go to maven->update project
+         
+        4.Then run your app,
+                right click on project->run as->spring boot app
+
+Steps for intellij: 
```

Comment:
Why should we have IDE specific settings What if we switch to another IDE?

Reply:
I have tested on sts and Intellij thats why i have mentioned to follow the steps for client perspective for bot IDE